src/components/vm/vmCloneDialog.tsx

```diff
@@ -57,10 +74,16 @@ export const CloneDialog = ({ name, connectionName }) => {
         }
 
         setInProgress(true);
-        const options = { pty: true };
-        if (connectionName === "system")
-            options.superuser = "try";
-        return cockpit.spawn(["virt-clone", "--connect", "qemu:///" + connectionName, "--original", name, "--name", newVmName, "--auto-clone"], options)
+        return cockpit.spawn(
+            [
+                "virt-clone", "--connect", "qemu:///" + connectionName,
+                "--original", name, "--name", newVmName,
+                "--auto-clone"
+            ],
+            {
+                pty: true,
+                ...(connectionName === "system" ? { superuser: "try" } : { })
```

Comment:
btw, we accept `superuser: null`, so this could be written more conveniently. Totally bikeshedding, of course.

Reply:
But only quite recently, with the introduction of SuperuserMode, no?